packages/kbn-es-query/src/es_query/__tests__/build_es_query.js

```diff
@@ -103,6 +137,25 @@ describe('build query', function () {
       expect(result).to.eql(expectedResult);
     });
 
+    it('should use the default time zone set in the Advanced Settings in queries and filters', function () {
+      const queries = [{ query: 'extension:jpg', language: 'kuery' }, { query: 'bar:baz', language: 'lucene' }];
+      const filters = [{ match_all: {}, meta: { type: 'match_all' } }];
+      const config = { allowLeadingWildcards: true, queryStringOptions: {}, dateFormatTZ: 'Africa/Johannesburg' };
+
+      const expectedResult = {
+        bool: {
+          must: [decorateQuery(luceneStringToDsl('bar:baz'), config.queryStringOptions, config.dateFormatTZ), { match_all: {} }],
+          filter: [toElasticsearchQuery(fromKueryExpression('extension:jpg'), indexPattern, config.dateFormatTZ)],
```

Comment:
@Bargs I don't follow. Where should I update the query to use a date field?

Reply:
Line 148 where `extension:jpg` is being passed to `fromKueryExpression`. Since that query does not use a date field, the timezone that's passed in never gets used.